content/en/security/cloud_security_management/vulnerabilities/_index.md

```diff
@@ -30,45 +30,92 @@ further_reading:
+| Time to deploy across your infrastructure | Minutes                                       | Hours to weeks                 |
+| Vulnerability prioritization              | Yes                                           | Yes, with runtime context      |
+| Vulnerability scanning frequency          | 12 hours                                      | Real-time                      |
+
+| Vulnerability detection scope             | Agentless                                     | Unified Datadog Agent          |
+|-------------------------------------------|-----------------------------------------------|--------------------------------|
+| Host and host image                       | OS packages and app packages, mapped to image | OS packages                    |
+| Container image                           | OS packages and app packages, mapped to image | OS packages                    |
+| Cloud provider                            | AWS, [Azure (Preview)][15]                    | AWS, Azure, GCP, on-prem, etc. |
+| Operating system                          | Linux                                         | Linux, Windows                 |
+| Serverless                                | AWS Lambda                                    | -                              |
+| Container registries                      | [Amazon ECR (Preview)][16]                    | -                              |
+
+For more information on compatibility, see [CSM Vulnerabilities Hosts and Containers Compatibility][13]. If you need any assistance, see the [troubleshooting guide][14], or reach out to support@datadoghq.com.
+
+## Continuously detect, prioritize, and remediate exploitable vulnerabilities
+The [CSM Vulnerabilities Explorer][1] helps you investigate all vulnerabilities detected across container images, host images, running hosts, and serverless functions you own, using filtering and grouping capabilities.
+
+Focus on exploitable vulnerabilities first, using the Datadog Severity Score, combining the base CVSS score with many risk factors, including sensitive data, environment sensitivity, exposure to attacks, exploit availability, or threat intelligence sources.
+
+The [CSM Vulnerabilities Explorer][1] helps you address vulnerabilities that have fixes available, and provides guided remediation steps, helping Dev and Ops teams fix vulnerabilities faster. Triage, mute, comment, and assign vulnerabilities to easily manage their lifecycle.
+
+{{< img src="security/vulnerabilities/csm-vm-explorer-actionability.png" alt="The CSM Vulnerability Explorer displaying a vulnerability and the actions a user can take to remediate it" width="100%">}}
+
+## Automation and Jira integration
+Make CSM Vulnerabilities part of your daily workflow by setting up [security notification rules][17]:
+- Get alerted upon detection of an exploitable vulnerability for your scope
+- Automatically create Jira tickets
+- Configure SLAs to remediate vulnerabilities
```

Comment:
Do you configure the SLAs in the notification rules? I don't see where you would do so in the UI.

Reply:
You're right, SLA will be part of automation pipelines (curently in preview)